src/main/java/uk/gov/hmcts/darts/dailylist/mapper/CitizenNameMapper.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.darts.dailylist.mapper;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.dailylist.model.CitizenName;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class CitizenNameMapper {
+
+    private static final String NAME_DELIMITER = " ";
+
+    private static final int FORENAME_INDEX = 0;
+
+    private static final int SURNAME_INDEX = 1;
+
+    public CitizenName getCitizenName(String name) {
+        String[] citizenName = name.split(NAME_DELIMITER);
+        CitizenName retCitizenName = new CitizenName();
+
+        if (citizenName.length == 2) {
```

Comment:
Is `> 2` ever possible?

Reply:
nope the creation of the name is also controlled by this class which takes the forename and surname and adds a space between them.

Reply:
yep I looked at this again and the add case api does support adding  names of a non descript length (although the api should probably use citizen name for consistency. I have changed the code so it now consider surnames that consist of multiple values. This will avoid false positive matching when processing the daily list.